tools/data-model-exporter/data_model_exporter/dmExporter.py

```diff
@@ -136,10 +152,14 @@ def main():
     # write one file per class provided
     for key in json_dict:
         out_file_name = f"{key}.json"
-        with open(out_file_name, 'w') as f:
-            logging.info(json.dumps(json_dict[key], indent=4))
-            json.dump(json_dict[key], f)
-
+        if(path.exists(out_file_name)):
+            rewrite = input(out_file_name + "already exists. Rewrite? (y/n)")
+            if rewrite == "y":
+               write_to_json(out_file_name,json_dict,key)
+            else:
+                print("not overwriting " + out_file_name)
```

Comment:
⛏️ I don't think this additional `else` + logging statement is necessary, the program can just exit (i.e. try running `rm -i foo.txt` from your command line and observe its behavior.

Reply:
the logging statement can be omitted, but since this branch is part of a loop, it definitely shouldn't be exiting here

Reply:
(also note that you should favor using `logging` rather than `print`, e.g. `logging.info`)